docs/CHANGELOG.md

```diff
@@ -20,7 +20,8 @@ The format of this change log follows the advice given at [Keep a CHANGELOG](htt
 - Updated project dependencies to current [moodle-local_moodlecheck](https://github.com/moodlehq/moodle-local_moodlecheck) and [moodle-local_ci](https://github.com/moodlehq/moodle-local_ci) versions.
 
 ### Added
-- Detect existence of legacy php-webdriver, and use a different Firefox image when it is in use
+- Detect existence of legacy php-webdriver, and use a different Firefox image when it is in use.
+- GitHub Actions use manual and example.
```

Comment:
Looking to other documentation additions in the CHANGELOG... I've see that in the past we have added a link to the articles from here... maybe we should do the same?

Reply:
Yep, good idea.

Reply:
Done.